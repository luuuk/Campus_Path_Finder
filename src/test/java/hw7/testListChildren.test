# This test is checks the ability of a Node to list its children

LoadGraph test test.tsv

ListChildren test a

AddEdge test a b 1.0

ListChildren test a

AddEdge test a b .1

ListChildren test a