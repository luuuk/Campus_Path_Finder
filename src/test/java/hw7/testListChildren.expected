# This test is checks the ability of a Node to list its children

loaded graph test

the children of a in test are: b(0.500) c(0.333) e(1.000)

added edge 1.000 from a to b in test

the children of a in test are: b(0.500) b(1.000) c(0.333) e(1.000)

added edge 0.100 from a to b in test

the children of a in test are: b(0.100) b(0.500) b(1.000) c(0.333) e(1.000)
